README.md

```diff
@@ -10,12 +10,10 @@ See [how to contribue](#how-to-contribute)
 
 ## Legends
 
-```
-🇬🇧 Bahasa Inggris
-🇮🇩 Bahasa Indonesia
-🌟 Dokumentasi Resmi dari [developer.android.com](developer.android.com)
```

Comment:
Ummm sorry but i forgot to remove the link. My intention is just to keep it to plain text